local-setup/github-action-config-rococo.json

```diff
@@ -0,0 +1,49 @@
+{
```

Comment:
Can we keep `local-setup/github-action-config-one.json`? It's still useful for local testing

Reply:
This is a redundant file. There is an identical one `github-action-config-one-worker.json`.